### ISSUE TITLE
Electrolysis support (Fennec 4+, future Firefox)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JSONovich/jsonovich#22]
```
Yet another nice-to-have...

Currently blocked by https://bugzilla.mozilla.org/show_bug.cgi?id=621406 - we 
may have to find something other than stream conversion if Electrolysis 
(separate UI/content/plugin processes) is eventually coming to desktop Firefox, 
since that's apparently why it won't work in Fennec.

```

Original issue reported on code.google.com by `W.Elwoo...@gmail.com` on 27 Jun 2011 at 2:30
